plugins/aap-backend/src/dynamic/alpha.ts

```diff
@@ -0,0 +1,40 @@
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import {
+  coreServices,
+  createBackendModule,
+} from '@backstage/backend-plugin-api';
+import { BackendDynamicPluginInstaller } from '@backstage/backend-plugin-manager';
+import { catalogProcessingExtensionPoint } from '@backstage/plugin-catalog-node/alpha';
+
+import { AapResourceEntityProvider } from '../providers';
+
+export const dynamicPluginInstaller: BackendDynamicPluginInstaller = {
+  kind: 'new',
+
+  install: createBackendModule({
+    moduleId: 'catalog-backend-module-aap',
+    pluginId: 'catalog',
+    register(env) {
+      env.registerInit({
+        deps: {
+          catalog: catalogProcessingExtensionPoint,
+          config: coreServices.rootConfig,
+          logger: coreServices.logger,
+          scheduler: coreServices.scheduler,
+        },
+        async init({ catalog, config, logger, scheduler }) {
+          catalog.addEntityProvider(
+            AapResourceEntityProvider.fromConfig(config, {
+              logger: loggerToWinstonLogger(logger),
+              schedule: scheduler.createScheduledTaskRunner({
+                frequency: { minutes: 30 },
+                timeout: { minutes: 3 },
+              }),
+              scheduler: scheduler,
```

Comment:
Currently this would be stuck using the 30 minute frequency schedule since schedule currently takes precedence over scheduler, since https://github.com/janus-idp/backstage-plugins/pull/827 hasn't been merged yet.

Reply:
Do we want to just remove the "default" schedules to ensure users can override at the moment?

Reply:
These alpha are mostly prepared for the time we'll be able to use the new backend system.
They are not wired for now, but brought here for completeness sake.

Reply:
What happens if the user doesn't provide any setting in the config ?

Reply:
More generally this is just a preparatory PR here.
These plugins will *not* be used as external dynamic plugins very soon, so maybe we can expect PR https://github.com/janus-idp/backstage-plugins/pull/827 to land in the meantime ?

Reply:
That would be ideal

Reply:
or issue a followup PR is the final choice is to only keep a required scheduler config through the appconfig